### ISSUE TITLE
Checks for DNS string

### ISSUE THREAD

**Issue by bengerman13** [cloud-gov/external-domain-broker-migrator#43]
The migrator should check for the yet-to-be-determined magic DNS string on the acme-challenge TXT record before beginning a migration

**Comment by ccostino** [comment]
We need to create the script to create the new migration records for these DNS strings on our side before we can check for their customer equivalents.  The general flow would look like this for our script:

- Get the list of instances from `CdnRoute`
- Get the domain names for each `CdnRoute` instance
- Create the TXT records in Route53 for each of those

With these TXT records in place, we could then be on the lookout for the corresponding customer CNAME record when they're ready for migration, which is what the check will be for.

**Comment by bengerman13** [comment]
I think that script is a different chunk of work from this one, though :)

**Comment by ccostino** [comment]
😆 fair enough!  I'll write a separate issue for it, thanks for calling that out. :-)

**Comment by ccostino** [comment]
Issue #58 created to account for the creation of the script first; moving this back to the backlog for now.

**Comment by ccostino** [comment]
@bengerman13 this DNS string check should be of the customer CNAME record that is added in, and checking (via regex, string split, etc.) a portion of it compared to the TXT record that we created with the semaphore, right?

**Comment by bengerman13** [comment]
given a CdnRoute with `domain_external` = `example.com` the migrator should query dns for a TXT record `_acme-challenge.example.com`. If the result is `cloud-gov-migration-ready`, it should consider the instance ready for migration

**Comment by jfredrickson5** [comment]
Resolved in #62.
